config/sepc/retina_r50v1b_fpn_pconv_1x.py

```diff
@@ -0,0 +1,249 @@
+from models.retinanet.builder import RetinaNet as Detector
+from models.NASFPN.builder import ResNetV1bFPN as Backbone
+from models.sepc.builder import RetinaNetNeckWithBNWithSEPC as Neck
+from models.sepc.builder import RetinaNetHeadWithBNWithSEPC as RpnHead
+from mxnext.complicate import normalizer_factory
+
+
+def get_config(is_train):
+    class General:
+        log_frequency = 100
+        name = __name__.rsplit("/")[-1].rsplit(".")[-1]
+        batch_image = 4 if is_train else 1
+        fp16 = True
+
+
+    class KvstoreParam:
+        kvstore     = "nccl"
+        batch_image = General.batch_image
+        gpus        = [0, 1, 2, 3, 4, 5, 6, 7]
+        fp16        = General.fp16
+
+
+    class NormalizeParam:
+        normalizer = normalizer_factory(type="syncbn",  ndev=len(KvstoreParam.gpus), wd_mult=1.0, lr_mult=1.0, eps=1e-4, mom=0.997)
+
+
+    class BackboneParam:
+        fp16 = General.fp16
+        normalizer = NormalizeParam.normalizer
+        depth = 50
+
+    class NeckParam:
+        fp16 = General.fp16
+        normalizer = NormalizeParam.normalizer
+
+    class SEPCParam:
+        out_channels=256
+        Pconv_num=4
+        start_level=1
+        pconv_deform=False 
+        iBN=None
```

Comment:
ibn = False

Reply:
This implementation handles case with ibn = None or False, they are equivalent.